Formula/licensee.rb

```diff
@@ -0,0 +1,123 @@
+class Licensee < Formula
+  desc "Tool to detect under what license a project is distributed"
+  homepage "https://licensee.github.io/licensee/"
+  url "https://github.com/licensee/licensee/archive/v9.14.1.tar.gz"
+  sha256 "9f90024b4b8de833349c1365ada86657bc6dc25c4cce15c6e5230883a6d39294"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "libgit2"
+  depends_on "ruby"
```

Comment:
Did you try uses_from_macos instead?

Reply:
Yes, the initial commit had uses_from_macos